File: floor/floor.java
public class floor {
    // ATOMIC_FUNCTION:  floor(val)
    // TAGS: math，numeric
    // LANG:  java
    // FUNCTION: floor(val)



    public static void main(String args[]) {
        double start_val = 1.2;
        double expected_end_val = 1;

        double end_val = Math.floor(start_val);

        // TEST
        if(end_val == expected_end_val) {
            System.out.println("PASS");
        } else {
            System.out.println("FAIL");
        }
    }
}

File: floor/floor.py
#!/usr/bin/env python
# coding: utf-8

# In[2]:


# ATOMIC_FUNCTION:  floor(val)
# TAGS: math，numeric
# LANG:  python
# FUNCTION: floor(val)
import math
start_val = 1.2;
expected_end_val = 1;

end_val = math.floor(start_val);



# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")


# In[ ]:




